### DISCUSSION TITLE
Markdown formatting outside of section

### DISCUSSION THREAD

**Discussion by ChrisButterworth** [tighten/jigsaw#615] [Q&A] [answered]
Looking to use Markdown in an additional field but no html formatting found when outputting the field.

Any advice would be great.

**Comment by bakerkretzmar** [comment, marked as answer]
By "field" do you mean you have some markdown in your frontmatter? Like this?

```mdx
---
description: |
  ## Subheading here
  
  More markdown description content.
draft: true
---
# Title

Main content.
```

I think you should be able to handle that by doing something like this in your template:

```blade
<div>
    {!! \Illuminate\Container\Container::getInstance()->make('markdownParser')->parse($page->description) !!}
</div>
```

Jigsaw can't really handle that kind of thing for you because we have no way of knowing if there's markdown in your frontmatter or not.
